docs/BUILDER_INTEGRATION.md

```diff
@@ -202,6 +202,12 @@ When using the builder in an app with server side persistence, the following rou
 
 Auth is up to the implementer for now
 
+### Implement a server for the upload route `/audiofiles/upload`
```

Comment:
Just wondering why the existing dev server wasn't sufficient? Could you also link to the documentation of the server endpoint contract in `routes.md` for this?